docs/fees_and_rewards/Dynamic_Issuance.md

```diff
@@ -0,0 +1,278 @@
+
+
+
+## Issuance Function
+
+The issuance function currently consists of two components: a decaying component and a utilization-based component. The decaying component issues a constant reward for some initial interval and gradually reduces it over time. The utilization-based component issues a reward based on the average utilization of blockspace in recent blocks.
+Such construction allows for more decaying components to be included for additional incentivization.
+
+### Full  Block
+
+`MAX_NORMAL_BLOCK_LENGTH` is the byte size limit of a full block set to consensus and equal to `0.75*MAX_BLOCK_LENGTH` (currently 3.75 MiB for) normal txs (different from operational (e.g., votes, equivocation) and mandatory (e.g., new archived segments))
+
+
+### Utilized Block Resources
+
+`AvgBlockspaceUsage` (in bytes) 
+
+Item that stores the utilization of blockspace (in bytes) by the normal extrinsics used to adjust issuance. Should be ≥0 and ≤`MAX_NORMAL_BLOCK_LENGTH`. 
+Blockspace utilization is computed as an exponential moving average of blockspace utilization of the last `num_blocks` blocks, updated every block.
+
+`num_blocks`: number of the blocks over which utilization is measured, should be ≥0 (Currently, 100)
+
+1. Compute the amount of blockspace utilized in this block `used_blockspace` ****as the sum of encoded lengths of normal transactions in block body.
+2. For genesis block `AvgBlockspaceUsage = used_blockspace`=0
+3. If `num_blocks==0` , return the `used_blockspace` of the current block.
+4. Else if `block_height <= num_blocks`, store new value 
+`(AvgBlockspaceUsage + used_blockspace)/2`
+5. After `block_height > num_blocks`, use `multiplier = 2/(num_blocks+1)`
+Store new value 
+`multiplier * used_blockspace + (1-multiplier) * AvgBlockspaceUsage`
+
+
+### Issuance Components
+
+We define a base block issuance as the number of tokens issued per block in a static case: a fixed reward to block proposer and `EXPECTED_VOTES_PER_BLOCK` voters.
+
+`RemainingIssuance`: tokens left to issue to farmers at any given time
+
+We define a reward split rule where the block proposer and voters are entitled to a share of the base reward:
+
+- `proposer_share = 1`
+- `voters_share`, currently `=EXPECTED_VOTES_PER_BLOCK` (for gemini-3h)
+- `total_shares = proposer_share + voters_share`
+
+The total actual reward issued includes a sum of several independent issuance components, each characterized by different values of common parameters based on the value of `BASE_REWARD` computed off-chain beforehand as described in [Off-chain Issuance curve parameters setting](docs/fees_and_rewards/Dynamic_Issuance#off-chain-issuance-curve-parameters-setting).  
+
+$B$ is the block starting from which the rewards component is activated. It can be set manually by an extrinsic initializing the component or automatically if rewards are subject to [Space Race](docs/fees_and_rewards/Dynamic_Issuance#space-race) and stored in `RewardsEnabled` runtime item.
```

Comment:
I realized this was not accurate.
```suggestion
$B$ is the block starting from which the rewards component is activated. It can be set manually by an extrinsic initializing the component or automatically if rewards are subject to [Space Race](#space-race).
```